build/compile/LzmaArchive.targets

```diff
@@ -115,11 +115,7 @@
       </GenerateNuGetPackagesArchiveVersion>
       <PropertyGroup>
         <NugetPackagesArchiveName>nuGetPackagesArchive.$(NuGetPackagesArchiveVersion).lzma</NugetPackagesArchiveName>
```

Comment:
I couldn't remove the rest of these because they depend on the GenerateNuGetPackagesArchiveVersion task. The task looks complicated and I wasn't sure if I could compute that value without a task.